python-avd/pyavd/_cv/client/models.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2025 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from dataclasses import dataclass
+from typing import Literal
+
+from typing_extensions import Self
+
+from pyavd._cv.api.arista.tag.v2 import ElementType, Tag, TagAssignment
+
+STRING_TO_ELEMENT_TYPE_MAP = {
+    "device": ElementType.DEVICE,
+    "interface": ElementType.INTERFACE,
+    "unspecified": ElementType.UNSPECIFIED,
+}
+
+ELEMENT_TYPE_TO_STRING_MAP = {
+    ElementType.DEVICE: "device",
+    ElementType.INTERFACE: "interface",
+    ElementType.UNSPECIFIED: "unspecified",
+}
+
+
+@dataclass(frozen=True)
+class CVTag:
+    """
+    Represent the input model for a CloudVision Tag.
+
+    Attributes:
+        element_type: The type of network element the tag applies to.
+        label: The label of the tag.
+        value: The value of the tag.
+    """
+
+    element_type: Literal["device", "interface", "unspecified"]
+    label: str
+    value: str
+
+    def to_element_type(self) -> ElementType:
```

Comment:
After reading more in the code, let's call this method `get_element_type` since we are just taking that one field and converting it. Also add a docstring.